indico/core/signals/__init__.py

```diff
@@ -8,6 +8,7 @@
 from indico.core.signals import (acl, agreements, attachments, category, event, event_management, menu, plugin, rb, rh,
                                  users)
 from indico.core.signals.core import *
+from indico.core.signals.lowlevel import *
```

Comment:
I would keep it as a module import so it's explicit via `indico.core.signals.lowlevel.*`